src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+class ColorSupplier {
+
+    public static Color getRandomColor() {
+        Color[] colors = Color.values();
+        Random random = new Random();
```

Comment:
Please extract random to class variable in order to avoid new object creation each time method is called